app/flame_chart/profile_model.ts

```diff
@@ -85,8 +90,21 @@ export function buildThreadTimelines(events: TraceEvent[], { visibilityThreshold
       event.dur &&
       event.dur > visibilityThreshold
   );
-  events.sort(eventComparator);
+  
+  for (const [tid, threadName] of threadNameByTid.entries()) {
+    if (tidByThreadName.has(threadName)) {
```

Comment:
Do we need this if/else? Doesn't seem like we need the minimum tid, any old one will do.

Maybe you can also update `tidByThreadName` to be something like `lastTidByThreadName`, or as I like to name my maps `threadNameToLastTidMap`.

Reply:
@bduffany was there a specific reason we discussed keeping the minimum tid?

Reply:
There was, but it's no longer applicable because it turned out that Bazel assigns those TIDs in an order that doesn't really match the thread names -- so we decided to sort by thread name instead